k8s.gcr.io/images/k8s-staging-sig-storage/images.yaml

```diff
@@ -82,6 +82,9 @@
   dmap:
     "sha256:c1bedac8758029948afe060bf8f6ee63ea489b5e08d29745f44fab68ee0d46ca": [ "v2.2.2" ]
     "sha256:2de1d15fc1f2a33618c61ff8d499ef22a7b2b8961952f059656a245b98498ba4": [ "v3.0.0" ]
+- name: nfs-subdir-external-provisioner
```

Comment:
This file is generated by https://github.com/kubernetes/k8s.io/blob/d928dcfbd5f01e9c0cf667f02f094c5e77b8191d/k8s.gcr.io/images/k8s-staging-sig-storage/generate.sh#L18. Can you add the repo to the list and then run the script?

Reply:
done, also sorted it alphabetically.